### PR TITLE
created CharacterServiceImpl, started the Character model, stated the…

### LINKED ISSUE #36: [DEV] UserCharacter Model
Currently skeleton was created for testing purposes, but you will need to update this....

### LINKED ISSUE #38: [DEV] Character Service

### PR DESCRIPTION
**Pull request by FullSol**
Completed the following in this PR

1. Created CharacterServiceImpl
2. Started the Character model
3. Started the CharacterRepository

closes #38

### CODE REVIEW

**Review by pyaeger — APPROVED** [review]
Thank you and approved.

File: src/main/java/com/amplifier/services/CharacterServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class CharacterServiceImpl implements CharacterService {
+
+    @Autowired
+    private CharacterRepository characterRepository;
+
```
Comment (L3viathon) [inline src/main/java/com/amplifier/services/CharacterServiceImpl.java:18]: `@Override` annotation for implementing CharacterService interface methods?

File: src/main/java/com/amplifier/services/CharacterServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    public boolean create(Character character) {
+        int pk = characterRepository.save(character).getId();
+        return (pk > 0) ? true : false;
+    };
+
```
Comment (L3viathon) [inline src/main/java/com/amplifier/services/CharacterServiceImpl.java:24]: `@Override` annotation for implementing CharacterService interface methods?

File: src/main/java/com/amplifier/services/CharacterServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    public Character findById(int id) {
+        return characterRepository.findById(id);
+    };
+
```
Comment (L3viathon) [inline src/main/java/com/amplifier/services/CharacterServiceImpl.java:29]: `@Override` annotation for implementing CharacterService interface methods?

File: src/main/java/com/amplifier/services/CharacterServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    public List<Character> findAll() {
+        return characterRepository.findAll();
+    };
+
```
Comment (L3viathon) [inline src/main/java/com/amplifier/services/CharacterServiceImpl.java:34]: `@Override` annotation for implementing CharacterService interface methods?

File: src/main/java/com/amplifier/services/CharacterServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    public boolean update(Character character) {
+        Character target = characterRepository.findById(character.getId());
+        target.setName(character.getName());
+        target.setRealm(character.getRealm());
+        return (characterRepository.save(target) != null) ? true : false;
+    };
+
```
Comment (L3viathon) [inline src/main/java/com/amplifier/services/CharacterServiceImpl.java:42]: `@Override` annotation for implementing CharacterService interface methods?

File: src/main/java/com/amplifier/services/CharacterServiceImpl.java (hunk 2 above)
Comment (L3viathon) [reply src/main/java/com/amplifier/services/CharacterServiceImpl.java:24]: method name `getById`?

File: src/main/java/com/amplifier/services/CharacterServiceImpl.java (hunk 3 above)
Comment (L3viathon) [reply src/main/java/com/amplifier/services/CharacterServiceImpl.java:29]: method name `getAll`?

**Review by L3viathon — CHANGES_REQUESTED** [review]

**Review by L3viathon — APPROVED** [review]
Great work mister Calvin!

**Pull request merged**